sdk/cosmosdb/cosmos/review/cosmos.api.md

```diff
@@ -595,11 +681,12 @@ export const Constants: {
     };
     AllVersionsAndDeletesChangeFeedWireFormatVersion: string;
     ChangeFeedIfNoneMatchStartFromNowHeader: string;
+    DefaultEncryptionCacheTimeToLive: number;
 };
 
 // @public
 export class Container {
-    constructor(database: Database, id: string, clientContext: ClientContext);
+    constructor(database: Database, id: string, clientContext: ClientContext, encryptionManager?: EncryptionManager, _rid?: string);
```

Comment:
This constructor needs to be marked hidden/internal. Users cannot instantiate this class as `clientContext` is internal class.

Reply:
done